app/models/text_field_type.rb

```diff
@@ -20,6 +22,22 @@ def mapping
 
   private
 
+  def less_than
+     errors.add(:text, "must be less_than #{@validations[:max]}") if @integer <= validations[:max]
+  end
+
+  def greater_than
+     errors.add(:text, "must be greater_than #{@validations[:min]}") if @integer >= validations[:min]
+  end
+
+  def validate_max?
+    @validations.key? :max
```

Comment:
JK - just saw your IntegerFieldType
delete all of this from `TextFieldType` please